apps/documentation/src/app/components/example/example.ts

```diff
@@ -524,6 +552,26 @@ node_modules`,
     "strictTemplates": true
   }
 }`,
+        ...(isTailwind
+          ? {
+              'tailwind.config.js': `/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./src/**/*.{html,ts}",
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [require('@tailwindcss/typography')],
```

Comment:
I don't think any of our examples use the typography plugin so we can remove it.

```suggestion
  plugins: [],
```